R/Find_MisSegregationRate_Surrogate.R

```diff
@@ -183,11 +183,17 @@ inp$Lagging.Chromosome = log(inp$Lagging.Chromosome)
 m = lm(inp$Lagging.Chromosome ~inp[[poi]])
 summary(m)
 ## Visualize
-pdf("~/Downloads/MS_predictions_9Cancers_TPM.pdf")
-plot(inp[[poi]],exp(inp$Lagging.Chromosome),col=3+inp$metastasis,xlab=poi,pch=20,cex=2,main=paste("R^2 =",summary(m)$adj.r.squared))
+## Set up colors
+dd <- unique(MN$cell.line)
+dd.col <- wes_palette(n=length(dd), "Darjeeling2", type = "discrete")
+names(dd.col)  <- dd
+pdf("~/Downloads/MS_predictions_9Cancers_TPM_.pdf")
+plot(inp[[poi]],exp(inp$Lagging.Chromosome),col=dd.col,xlab=poi,pch=ifelse(inp$metastasis>0, 8,20),cex=2,main=paste("R^2 =",summary(m)$adj.r.squared))
```

Comment:
need to index the colors with the name of the cell line orelse they will essentially be random. Sth like: `dd.col[inp$cell.line]`. In order to do that you need to add `cell.line` to `inp` when the object is instantiated.

Reply:
Updated in 04699c1

Reply:
Improved further per our Slack convo in 3e6e9ee